daemon/container.c

```diff
@@ -2157,8 +2160,21 @@ int
 container_add_net_iface(container_t *container, const char *iface, bool persistent)
 {
 	ASSERT(container);
+
+	IF_FALSE_RETVAL(cmld_netif_phys_remove_by_name(iface), -1);
```

Comment:
handle case that if is plugged in later?

Reply:
yes makes sens but this would need more effort, se some comment above (uevent subsystem)